zuul-core/src/main/java/com/netflix/zuul/filters/ZuulServletFilter.java

```diff
@@ -109,7 +113,7 @@ void init(HttpServletRequest servletRequest, HttpServletResponse servletResponse
     void error(ZuulException e) {
         RequestContext.getCurrentContext().setThrowable(e);
         zuulRunner.error();
-        e.printStackTrace();
+        LOG.error(e.getMessage(), e);
```

Comment:
My preference would be to remove `e.printStackTrace()` altogether and let the 'error' filters determine if they want to log the error.